src/app/src/pages/redteam/setup/components/StrategyConfigDialog.tsx

```diff
@@ -542,7 +549,7 @@ export default function StrategyConfigDialog({
 
   return (
     <Dialog open={open} onClose={onClose} maxWidth="sm" fullWidth>
-      <DialogTitle>Configure {strategy}</DialogTitle>
+      <DialogTitle>Configure {strategyData?.name ?? strategy}</DialogTitle>
```

Comment:
What does this change?

Reply:
This fixes a bug where strategy ids are displayed instead of their names.